content/docs/start/bitbucket.md

```diff
@@ -38,28 +35,27 @@ all the supported CI systems.
              - python train.py
 
              - cat metrics.txt > report.md
-             - cml-publish plot.png --md >> report.md
-             - cml-send-comment report.md
+             - cml publish plot.png --md >> report.md
+             - cml send-comment report.md
    ```
 
-5. In your text editor of choice, edit line 15 of `train.py` to `depth = 12`.
+4. In your text editor of choice, edit line 16 of `train.py` to `depth = 5`.
```

Comment:
Is this line number correct?

Reply:
I assume the code is/should be the same as the GitHub example? Why are they different?

Reply:
https://bitbucket.org/iterative-ai/example-cml/src/feed5ba31b44ac1fd46cb4ee411eca77b14b978a/train.py#lines-15

Reply:
> _Why are they different?_

Because they haven't been generated programmatically and, as such, are prone to staleness and drift? 😈 Linking to https://github.com/iterative/example-repos-dev/issues/100 for future references.

Reply:
looks like the repos are in sync now, so it's meant to be "line 15" again?

Reply:
Yes, I guess so